facilities/views.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import generics
+from .models import Owner, Facility, Service
+
+
+from .serializers import (
+    OwnerSerializer, ServiceSerializer, FacilitySerializer
+)
+
+
+class OwnerListView(generics.ListCreateAPIView):
+    queryset = Owner.objects.all()
+    serializer_class = OwnerSerializer
+    filter_fields = ()
```

Comment:
The models are few enough that we can do filter classes for everything.

At first blush, that appears to be overkill; it is not ( will explain later ).